src/main/java/Seconds.java

```diff
@@ -1,12 +0,0 @@
-public class Seconds {
-
-
-
-   public static int changeMinutesToSeconds (int minutes)  {
-
-           int seconds = minutes * 60;
```

Comment:
eee, wszystkie zadania finalnie powinny zostać w tym repo :)
Wrzuć je spowrotem, po prostu do jakieogś osobnego pakietu np /zad3/ (czy które tam to było)